libcxx/test/configs/cmake-bridge.cfg.in

```diff
@@ -23,8 +23,8 @@ config.recursiveExpansionLimit = 10
 config.test_exec_root = os.path.join('@CMAKE_BINARY_DIR@', 'test')
 
 # Add substitutions for bootstrapping the test suite configuration
-import shlex
-config.substitutions.append(('%{cxx}', shlex.quote('@CMAKE_CXX_COMPILER@')))
+cxx = libcxx.test.format.cmake_workaround_for_cxx_on_Apple(shlex.quote('@CMAKE_CXX_COMPILER@'))
```

Comment:
Can we please move this from `cmake-bridge.cfg.in` where it affects all `.cfg.in` files, even the user-supplied ones, to `apple-*-backdeployment.cfg.in` and `apple-*-shared.cfg.in`?